packages/docs/docs/tutorial/get-started/create-and-integrate-the-first-app.mdx

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 2
+---
+
+import Columns from '@components/Columns';
+
+# Create and integrate the first application
+
+Click the "Create" button on the right, and the browser will redirect to the Application tab. Then click the "Create Application" button on this page.
+
+**Step 1**
+
+In the opening modal, choose which type of application you'd like to integrate:
+
+- **Native App** is an app that runs in a native environment, e.g., iOS app, Android app.
+- **Single Page App** is an app that runs in a web browser, which updates the page with the new data from the server without loading entire new pages. E.g., React DOM app, Vue app.
+- **Traditional Web App** is an app that renders and updates pages by the web server alone. E.g., JSP, PHP.
+
+We prepared different tutorials for each application type, and it'll be helpful if you choose the type properly.
+
+**Step 2**
+
+Enter the application name, e.g., "Bookstore," and click "Create Application."
+
+**Step 3**
+
+Ta-da! You just created your first application in Logto. You'll see a congrats page which includes a detailed integration guide. Follow the guide to see what the experience will be in your application.
+
+Meanwhile, Logto offers some official SDKs to bring the integration experience to another level (introduction included in the guide). You can also pick the guide from the list below if you skip the page by accident:
+
+<Columns columns={[
+  { title: 'Native App', items: [{ key: 'iOS' }, { key: 'Android' }] },
+  { title: 'Single Page App', items: [{ key: 'Vanilla JS' }, { key: 'React' }, { key: 'Vue' }] },
+  { title: 'Tranditional Web App', items: [{ key: 'ExpressJS' }, { key: 'Other Web Servers' }] },
+]} />
```

Comment:
I think we need to emphasis that the user should use the ID of the application he/she just created in admin console as the value of `clientId` in the `LogtoConfig`

Reply:
agreed

Reply:
actually this should be covered in guide?

Reply:
in specific guide?

Reply:
I think i'll be cool since the App ID exposes in the table and header - big enough. and we'll mention it in every integration guide as well

Reply:
OK, let's move on